### DISCUSSION TITLE
TuringCourses/major/ai_basic/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#100] [General]
# TuringCourses/major_basic/ai_basic/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_basic/ai_basic/

**Comment by HobbitQia** [comment]
杨易老师给 21 级的同学上过一次课，说话幽默讲课有趣，循序渐进，据说比较关心学生。但考虑他在 2023 有挂名的情况，各位不要抱太大希望。（可能挂名的是杨老师，但是上课的是他手下的学生[ac01]）
章敏老师以前带的强数班，老师本身水平很高（计算机和数学双博士），据说人比较好。

个人认为在机器学习理论方面，这门课不如隔壁数院的数据建模与分析，具体介绍可见其他课程部分。

**Comment by lhxcs** [comment]
章敏老师班今年分数构成是5%签到+35%lab+60%期末。虽然老师水平很高，但是上课基本是在念PPT，每节到课人数都是个位数。但尽管如此整个学期老师没有点过名，最后也是很善良地把签到分都送给我们了。
从最后的给分来看，章敏老师平时那40%基本是给满了，最后的成绩主要取决于期末的60%。

个人认为这门课基本学不到什么东西，开成像网安导那样的讲座形式似乎更合理些。从应试的角度，由于考试内容不会超出PPT，而且大题都是PPT上的例子，因此考前可以过一遍PPT，熟悉一下概念，有时间的话PPT上的例题也可以跟着算一遍。

**Comment by dieKarotte** [comment]
挂名yy老师的课程实际上是由yy老师和朱霖潮老师共同授课完成的，虽然85%的课程是由zlc老师上的（x 。上课认真听的话可以感受到zlc老师还是很有水平并且能很好的把课程讲清楚的，可惜这门课的课程内容算不上合理，也只考察对于ppt内容的认真背诵。在zlc老师的课上回答问题可以得到一些可以吃的reward，也算是老师想把课上好的一点点表示吧，可惜1:25-3:50的课程我们都昏昏欲睡/翘了。考前背诵ppt，过一遍老师复习课上的关键内容就差不多可以拿到被宽宥的分数。
